src/containerapp/azext_containerapp/custom.py

```diff
@@ -2642,6 +2647,52 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+def create_or_update_java_logger(cmd, logger_name, logger_level, name, resource_group_name, no_wait=False):
+    raw_parameters = locals()
+
+    java_logger_decorator = JavaLoggerDecorator(
+        cmd=cmd,
+        client=ContainerAppPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    return java_logger_decorator.create_or_update_logger()
```

Comment:
Recommend refer `ContainerappCreateDecorator`, separate your logic in to 
```
validate_arguments()
construct_payload()
create()
```

Reply:
updated to 

```
validate_enabled_java_agent()
create_or_update()
```

construct_payload seems to be done in decorator is better than in custom.py.

Reply:
> Recommend refer `ContainerappCreateDecorator`, separate your logic in to
> 
> ```
> validate_arguments()
> construct_payload()
> create()
> ```

updated to 

```
validate_enabled_java_agent()
create_or_update()
```

construct_payload seems to be done in decorator is better than in custom.py.